src/interfaces/highs_c_api.h

```diff
@@ -146,6 +162,14 @@ int Highs_getHighsDoubleOptionValue(void* highs,  //!< HiGHS object reference
                                     double* value        //!< value of option
 );
 
+/*
+ * @brief
+ */
+int Highs_getHighsStringOptionValue(void* highs,  //!< HiGHS object reference
+                                    const char* option,  //!< name of the option
+                                    char* value          //!< pointer to allocated memory to store value of option
```

Comment:
Given the potential for a buffer overflow, I think there should be a big warning on this function. Alternatively provide a method to query the length of the string before querying its value.